pkg/realizer/pipeline/realizer.go

```diff
@@ -41,7 +42,8 @@ func NewRealizer() Realizer {
 type pipelineRealizer struct{}
 
 type TemplatingContext struct {
-	Pipeline *v1alpha1.Pipeline `json:"pipeline"`
+	Pipeline *v1alpha1.Pipeline      `json:"pipeline"`
+	Selected *map[string]interface{} `json:"selected"`
```

Comment:
I think we can go with just `map[string]interface{}` as maps are pointers to some internal runtime stuff (still, pointers)